test/image/mocks/zzz_smith_axes.json

```diff
@@ -0,0 +1,81 @@
+{
+  "data": [
+    {
+      "type": "scattersmith",
+      "re": [0, 0.1, 0.5, 1, 2, 3, 4, 5, 10, 0.1, 0.5, 10],
+      "im": [0, 0.1, 0.5, 1, 2, 3, 4, 5, 10, -0.1, -0.5, -10],
+      "mode": "lines+markers"
+    },
+    {
+      "type": "scattersmith",
+      "re": [0, 0.1, 0.5, 1, 2, 3, 4, 5, 10, 0.1, 0.5, 10],
+      "im": [0, 0.1, 0.5, 1, 2, 3, 4, 5, 10, -0.1, -0.5, -10],
+      "mode": "lines+markers",
+      "subplot": "smith2"
+    },
+    {
+      "type": "scattersmith",
+      "re": [0, 0.1, 0.5, 1, 2, 3, 4, 5, 10, 0.1, 0.5, 10],
+      "im": [0, 0.1, 0.5, 1, 2, 3, 4, 5, 10, -0.1, -0.5, -10],
+      "mode": "lines+markers",
+      "subplot": "smith3"
+    },
+    {
+      "type": "scattersmith",
+      "re": [0, 0.1, 0.5, 1, 2, 3, 4, 5, 10, 0.1, 0.5, 10],
+      "im": [0, 0.1, 0.5, 1, 2, 3, 4, 5, 10, -0.1, -0.5, -10],
+      "mode": "lines+markers",
+      "subplot": "smith4"
+    }
+  ],
+  "layout": {
+    "smith": {
+      "domain": {
+        "x": [0, 0.46],
+        "y": [0.56, 1]
+      },
+      "imaginaryaxis": {
+        "showgrid": false,
+        "showticklabels": false,
+        "visible": true
+      }
+    },
+    "smith2": {
+      "domain": {
+        "x": [0, 0.46],
+        "y": [0, 0.44]
+      },
+      "realaxis": {
+        "showgrid": false,
+        "visible": true
+      }
+    },
+    "smith3": {
+      "domain": {
+        "x": [0.54, 1],
+        "y": [0.56, 1]
+      },
+      "realaxis": {
+        "showgrid": true,
+        "showticklabels": false,
+        "visible": true
+      }
+    },
+    "smith4": {
+      "domain": {
+        "x": [0.54, 1],
+        "y": [0, 0.44]
+      },
+      "imaginaryaxis": {
+        "visible": false
+      },
+      "realaxis": {
+        "visible": false
+      }
+    },
+    "showlegend": true,
+    "margin": {"t": 40, "b": 40, "r": 40, "l": 40},
```

Comment:
Here let's set `"paper_bgcolor": "lightgray"` to help improve visibility.

Reply:
6b0ed67dcb0591f0e2d13ba1b04b68e12c6fc39c

Reply:
08e4e63c9ab6daa05215a4db43ca4045febfa330